plugins/out_stackdriver/stackdriver.c

```diff
@@ -446,6 +447,41 @@ static int get_severity_level(severity_t * s, const msgpack_object * o,
     return -1;
 }
 
+static int pack_json_payload(bool operation_extracted, int operation_extra_size, 
+                            msgpack_packer* mp_pck, msgpack_object *obj)
+{
+    /* Specified fields include operation, sourceLocation ... */
+    int to_remove = 0;
+    if(operation_extracted && operation_extra_size == 0) {
+        to_remove += 1;
+    }
+
+    int ret = msgpack_pack_map(mp_pck, obj->via.map.size - to_remove);
+    if(ret < 0) {
+        return ret;
+    }
+    else {
+        msgpack_object_kv* kv = obj->via.map.ptr;
+        msgpack_object_kv* const kvend = obj->via.map.ptr + obj->via.map.size;
+        for(; kv != kvend; ++kv	) {
+            if (strncmp(OPERATION_FIELD_IN_JSON, kv->key.via.str.ptr, kv->key.via.str.size) == 0 
+                && kv->val.type == MSGPACK_OBJECT_MAP) {
+
+                if(operation_extra_size > 0) {
+                    msgpack_pack_object(mp_pck, kv->key);
+                    pack_extra_operation_subfields(mp_pck, &kv->val, operation_extra_size);
+                }
+                continue;
+            }
+            
+            ret = msgpack_pack_object(mp_pck, kv->key);
+            if(ret < 0) { return ret; }
```

Comment:
use braces on different lines: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions

Reply:
Done.